openpype/plugins/load/delivery.py

```diff
@@ -95,6 +95,12 @@ def __init__(self, contexts, log=None, parent=None):
         template_label.setCursor(QtGui.QCursor(QtCore.Qt.IBeamCursor))
         template_label.setTextInteractionFlags(QtCore.Qt.TextSelectableByMouse)
 
+        renumber_frame = QtWidgets.QCheckBox()
+
+        renumber_frame_value = QtWidgets.QSpinBox()
+        max_int = (1 << 32) // 2
+        renumber_frame_value.setRange(-max_int, max_int - 1)
```

Comment:
If the result is new frame start we should 1.) rename the variables from `renumber...` to `frame_start...` (or something similar) and 2.) not allow to use less than `0` -> which makes it invalid.
```suggestion
        renumber_frame_value.setRange(0, max_int - 1)
```